### PR TITLE
fixing code blocks

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: _posts/jeffrey_burt/2017-01-13-d3-brush.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ if(majorVersion === '3') {
 #### Method 2
 Because the API has changed so much, in a lot of cases you can simply check for the existence of the methods you need.
 
-For example, if the v3 method you need is `d3.scale.linear` and the v4 one is `d3.scaleLinear` you could do
+For example, if the v3 method you need is `d3.scale.linear` and the v4 one is `d3.scaleLinear` you could do
```
Comment (shanewilson) [inline _posts/jeffrey_burt/2017-01-13-d3-brush.md:43]: are these extra new lines?

Comment (Jephuff) [reply _posts/jeffrey_burt/2017-01-13-d3-brush.md:43]: Yeah, I'm wondering if the code blocks are breaking without a newline ( code blocks are working after headers, but not in the two places I had no new line. 

But I'm really shooting in the dark here, cause it all works locally :)

**Comment by SidSyrus** [comment]
You sould use `~~~` for code bock instead of ` ``` `
We are using [kramdown](https://kramdown.gettalong.org/) for our markdown engine and it has new [syntax](https://kramdown.gettalong.org/syntax.html#fenced-code-blocks) for code blocks.

[Why we are using Kramdown as our MD engine](https://help.github.com/articles/updating-your-markdown-processor-to-kramdown/)

**Comment by Jephuff** [comment]
@SidSyrus Is the blog not using kramdown when rendering locally? how do we make it render the same?

**Comment by cheapsteak** [comment]
@Jephuff @SidSyrus We can in fact use ``` for code blocks. I upgraded our version of Jekyll (and thus the version of Kramdown) to the one Github uses in my PR

edit: fun fact, it seems to have been added to Jekyll by the Github folks specifically for github pages :)

> With kramdown and Rouge as the new defaults, syntax highlighting on GitHub Pages should work like you'd expect it to work anywhere else on GitHub, with native support for backtick-style fenced code blocks right within the Markdown.

https://github.com/blog/2100-github-pages-now-faster-and-simpler-with-jekyll-3-0

**Comment by Jephuff** [comment]
We've updated the local version of kramdown to match what github uses, so it should make the preview a little more accurate 👍

**Review by Fgerthoffert — APPROVED** [review]
Thanks

**Pull request merged**